From ad063909301e20ac6c1cd12487a4c54a6c9c8ab6 Mon Sep 17 00:00:00 2001
From: Aleksandrs Vinarskis <alex.vinarskis@gmail.com>
Date: Wed, 7 May 2025 02:55:30 +0200
Subject: [PATCH] arm64: dts: qcom: x1e80100-slim7x: align regulators with AeoB
 specification

Set regulators as described in respective decompiled AeoB files from
Lenovo Yoga Slim 7x.

Signed-off-by: Aleksandrs Vinarskis <alex.vinarskis@gmail.com>
---
 arch/arm64/boot/dts/qcom/x1e80100-lenovo-yoga-slim7x.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/x1e80100-lenovo-yoga-slim7x.dts b/arch/arm64/boot/dts/qcom/x1e80100-lenovo-yoga-slim7x.dts
index beb8477641fa..727c1bf23cb0 100644
--- a/arch/arm64/boot/dts/qcom/x1e80100-lenovo-yoga-slim7x.dts
+++ b/arch/arm64/boot/dts/qcom/x1e80100-lenovo-yoga-slim7x.dts
@@ -636,9 +636,9 @@ camera@36 {
 
 		orientation = <0>; /* front facing */
 
-		avdd-supply = <&vreg_l7m_2p8>;
-		dvdd-supply = <&vreg_l2m_1p2>;
-		dovdd-supply = <&vreg_l4m_1p8>;
+		avdd-supply = <&vreg_l7b_2p8>;
+		dvdd-supply = <&vreg_l1m_1p2>;
+		dovdd-supply = <&vreg_l3m_1p8>;
 
 		port {
 			ov02c10_ep: endpoint {
-- 
2.45.2
